lib/php/utils/EnvironmentFetcher.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+
+namespace App\utils;
+
+//not used for now
+class EnvironmentFetcher
+{
+    public static function getJSONEnv(): array
+    {
+        $output = [];
+        exec("jq -n env", $output);
```

Comment:
We probably won't need that at all.
The environment variables in PHP should be accessible in `$_ENV` or `$_SERVER` which is way easier to exploit.